### ISSUE TITLE
Count could accept a formatting string

### ISSUE THREAD

**Issue by andybak** [matiboy/django-admin-methods#3]
eg:

```
Units = admin_methods.list.count('units', format=_('{} unit'), format_plural=_('{} units')))
```

**Comment by matiboy** [comment]
@andybak You mean the way we display in each row? Instead of only showing the number of items, it would show `0 units` `1 unit` `10 units` etc?

**Comment by andybak** [comment]
Yep.

**Comment by matiboy** [comment]
Closed in #5 
Will include in 0.1.6 release
